open-api/rest-catalog-open-api.yaml

```diff
@@ -61,6 +61,14 @@ security:
   - OAuth2: [catalog]
   - BearerAuth: []
 
+tags:
+  - name: views
+    description: Requires server to support views
```

Comment:
I'm not sure I agree with this wording. Maybe something more like "Required as part of view support"?

To me, the goal is to clarify what parts of this spec are optional and what are grouped together. Views are a great example: if a service supports views, it has to support everything tagged `views`.

Reply:
in that case what about `These endpoints are required when a server supports XYZ`? I think this should be clear that endpoints grouped in this category are required to be implemented when supporting XYZ.

Reply:
Yes, although I'm not sure whether the correct term is "route" or "endpoint" so I'd probably just leave it out. "Required as part of view support" would work.

Reply:
Ha, I just realized that my latest suggestion matches my original suggestion. I didn't do that on purpose.